views/messages/index.erb

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta name='viewport' content='widht=devide-width,initial-scale=1'>
+    <title>Chitter</title>
+    <link href='/styles.css' rel="stylesheet" type="text/css">
+  </head>
+
+  <body>
+    <header>
+      <div class='title container'>
+        <h1>Chitter</h2>
+      </div>
+
+      <div class='login container'>
+        <% if !@user %>
+          <form action='/users/new' method='get'>
+            <input type='submit' value='Sign up' />
+          </form>
+
+          <form action='sessions/new' method='get'>
+            <input type='submit' value='Log in' />
+          </form>
+        <% else %>
+          <form action='/sessions/destroy' method='post'>
+            <input type='submit' value='Log out' />
+          </form>
+        <% end %>
+      </div>
+    </header>
+
+    <main>
+      <div class='main-container'>
+        <div class='welcome'>
+          <% if @user %>
+            <h1>Welcome, <%= @user.email %>!</h1>
+          <% end %>
+
+          <h3>
+            <%= flash[:notice] %>
+          </h3>
+        </div>
+
+        <div class='messages'>
+          <% @messages.each do |message|%>
+            <ul>
+              <li class='message'>
+                <div class='message-content'><%= message.content %></div>
+                <div class='author'><p>By <%= message.author %></p></div>
+                <div class='time'><%= message.pretty_date %></div>
+              </li>
+            </ul>
+          <% end %>
+        </div>
+
+        <div class='post'>
+          <form action='/messages/post' method='post'>
+            <input type='text' name='content' placeholder='Type in your peep' class='message-content'/>
+            <input type='submit' value='Post' id='post-button' />
+          </form>
+        </div>
+
+      </div>
+    </main>  
+  </body>
+</html>
```

Comment:
Perfect indentation, easy to read code. Descriptive class names. Good use of div and main to structure the web app!